src/Model/Widget.php

```diff
@@ -260,15 +260,15 @@ public function getController()
             return $this->controller;
         }
 
-        foreach (array_reverse(ClassInfo::ancestry($this->class)) as $widgetClass) {
+        foreach (array_reverse(ClassInfo::ancestry(get_class($this))) as $widgetClass) {
             $controllerClass = "{$widgetClass}Controller";
             if (class_exists($controllerClass)) {
                 break;
             }
         }
 
         if (!class_exists($controllerClass)) {
-            throw new Exception("Could not find controller class for $this->classname");
+            throw new Exception("Could not find controller class for get_class($this)name");
```

Comment:
Don't think this will work as the classname won't get printed.

Reply:
Ha, that'll teach me for search/replacing